erikbenedict/README-generator
# Feature/license functions

## Test

- run application locally, and open integrated terminal from root
- run ``` node index.js ``` in the terminal
- complete all prompts, and specifically in the license prompt,  choose a license from the list that isn't 'Unlicensed'
- after completion, open exampleREADME.md and see that a README was generated based on user input
- see that the license of choice is represented in the license banner at the top of the README
- see that a 'License' section was generated, and has a notice explaining which license the application is covered under, and that the license link takes you to the license of choice's page
- run the program again and choose 'Unlicensed' from the license prompt
- see that the newly generated 'exampleREADME.md' does not show a banner, and does not have a 'License' section
